test/fakes/fake-network-client.ts

```diff
@@ -98,7 +99,17 @@ export class FakeNetworkClientResponses {
    * Construct a default getTx response.
    */
   public static defaultGetTxResponse(): GetTxResponse {
-    return new GetTxResponse()
+    const transactionResult = new TransactionResult()
+    transactionResult.setResult('tesSUCCESS')
+
+    const meta = new Meta()
```

Comment:
What is `meta`?

Reply:
See “Response Format” here: https://xrpl.org/tx.html

Meta is defined as “ Various metadata about the transaction.”. The protobufs are just mirroring the existing JSON API, so we accepted their structure at face value.